### ISSUE TITLE
If the selected test class doesn't exists, raise an error.

### ISSUE THREAD

**Issue by ignazioc** [plu/pxctest#22]
I wold like to run only the tests in one class, therefore I run:

    bundle exec rake test:unit[MyAwesomeTestClass]

A small issue happens if the class name is misspelled.  In that case the project is still built, and the output looks like:

```
UnitTests - iPhone 5 iOS 10.2 - Finished executing 0 tests after 0.002s. 0 Failures, 0 Unexpected
UnitTests - iPhone 5 iOS 9.0 - Finished executing 0 tests after 0.001s. 0 Failures, 0 Unexpected
Total - Finished executing 0 tests. 0 Failures, 0 Unexpected
```
It clearly says `Finished executing 0 tests` but I would rather prefer a big red error message :P

**Comment by plu** [comment]
Good point, I actually run into this myself from time to time, wondering what's happening :).

**Comment by plu** [comment]
Fixed via 1b8b21c8c9a9ef697f0b0109b2b30d69e9a859d7
